cloud: add storage usage impact notes for TiKV and TiFlash (#22514)
pingcap/docs: tidb-cloud/monitor-built-in-alerting.md

| Total TiDB node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiDB to reduce the CPU usage percentage of the current workload.|
| Total TiKV node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiKV to reduce the CPU usage percentage of the current workload. |
| Total TiFlash node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiFlash to reduce the CPU usage percentage of the current workload. |

> | TiKV storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiKV to increase your storage capacity. Storage usage of TiKV exceeding 80% can cause latency spikes, and higher usage might cause requests to fail. |

Reviewer: ```suggestion
| TiKV storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiKV to increase your storage capacity. The storage usage of TiKV exceeding 80% can cause latency spikes, and higher usage might cause requests to fail. |
```

> | TiKV storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiKV to increase your storage capacity. When the storage usage of TiKV exceeds 80%, latency spikes occur, and higher usage might cause requests to fail. |

Author: ```suggestion
| TiKV storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiKV to increase your storage capacity. When the storage usage of TiKV exceeds 80%, latency spikes might occur, and higher usage might cause requests to fail. |
```

> | TiFlash storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. Usage above 80% indicates disk exhaustion and may result in request errors.|

Reviewer 2: ```suggestion
| TiFlash storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. Usage above 80% indicates disk exhaustion and may result in adding new tiflash replica to table hangs.|
```

> | TiFlash storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. Usage of TiFlash above 80% indicates disk exhaustion and requests might fail.|

Reviewer: ```suggestion
| TiFlash storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. Storage usage of TiFlash exceeding 80% indicates disk exhaustion and requests might fail. |
```
